### PR TITLE
Add test for CA CRL database

### PR DESCRIPTION
**Pull request by edewata**
A new test has been added to verify that the CRL database in CA contains the correct CRL initially, after revocation, and after unrevocation.

https://github.com/dogtagpki/pki/wiki/CA-CRL-Database

### CODE REVIEW

**Review by ckelleyRH — APPROVED** [review]
LGTM!

File: .github/workflows/ca-crl-test.yml [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+              -D pki_cert_id_generator=random \
+              -D pki_request_id_generator=random \
+              -v
+
+          # update CRL immediately after each cert revocation
+          docker exec pki pki-server ca-config-set ca.crl.MasterCRL.alwaysUpdate true
+
+          # restart CA subsystem
+          docker exec pki pki-server ca-undeploy --wait
+          docker exec pki pki-server ca-deploy --wait
```
Comment (fmarco76) [inline .github/workflows/ca-crl-test.yml:70]: Just for my curiosity, in other scripts the restart is done with `docker exec pki pki-server restart --wait` but here you undeploy and deploy again. Is the result the same?

**Review by fmarco76 — APPROVED** [review]
LGTM

File: .github/workflows/ca-crl-test.yml (hunk 1 above)
Comment (edewata) [reply .github/workflows/ca-crl-test.yml:70]: Yes, in this case we're modifying CA subsystem configuration, so we need to restart it. `pki-server restart` will restart the entire server including all subsystems in it, but `pki-server ca-undeploy/deploy` will just restart the CA subsystem, so it's faster.

I think I'll create a `pki-server <subsystem>-restart` to call `<subsystem>-undeploy/deploy`, then update all tests to use that.

**Comment by edewata** [comment]
@ckelleyRH @fmarco76 Thanks!

**Pull request merged**